src/main/java/org/frc5587/bunnybots/Door.java

```diff
@@ -0,0 +1,31 @@
+package org.frc5587.bunnybots;
+
+import edu.wpi.first.wpilibj.Servo;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class Door extends Subsystem {
+    /* 
+    "servo" means "i save" in latin, which is entirely irrelevant 
```

Comment:
I'm going to let it slide for now, because we'll be going back and documenting all of this code later, but when we're writing the code for the build season, I'm not going to accept Pull Requests if stuff like this is in them (although I will admit that I laughed at this)